collections/_article/godot-showcase-somar.md

```diff
@@ -0,0 +1,101 @@
+
+### Did you use any existing toolkits/plugins in developing this experience and what was the motivation behind choosing these or building your own solution?
+
+<strong class="daniel">Daniel:</strong> Not really! The project doesn’t feature movement, and user input is limited, so building a custom solution was not a problem. Furthermore, keeping the project as simple as possible was one of my goals, both to prevent bugs and keep the code clean and easy to understand.
+
+### What would you like to see evolve within Godot's XR support that would have made building this project easier?
+
+<strong class="daniel">Daniel:</strong> There was a single bug or unexpected behavior that I found during development, but nothing that couldn’t be worked around. For this project, I believe Godot was the right choice and made things as easy as they could be.
+
+### I understand the project currently targets the Meta Quest headsets. What motivated this choice?
+
+<strong class="fred">Fred:</strong> The Meta Quest headset was chosen for the SOMAR project due to its strong popularity, excellent performance, and ease of use. As a standalone device, the Quest 3 delivers high-quality visuals and immersive experiences without needing a PC or external sensors, making it ideal for both museum venues and broader accessibility. Its widespread presence in the XR market ensures compatibility with future updates, allowing us to reach a wide audience while providing a seamless and engaging experience.
+
+### How was the experience of using Godot with the Meta Quest?
+
+<strong class="daniel">Daniel:</strong> Awesome! I used Meta Quest Link for most of the development to quickly test things, but every once in a while I deployed the project directly into the Meta Quest to measure performance. Fun fact: the project uses the Khronos OpenXR runtime, and it worked flawlessly first time.
+
+### I also understand the project is also capable of using the Monado OpenXR runtime. Can you tell me a bit more about the motivation and goals of this port?
+
+<strong class="fred">Fred:</strong> The motivation behind also targeting the Monado OpenXR runtime was to ensure broader accessibility and compatibility across different platforms, particularly for low-cost, phone-based VR systems. Monado’s open-source nature aligns with our goal of creating a flexible, cost-effective solution that’s not reliant on proprietary ecosystems. For this project, Collabora dedicated engineering resources to develop a new Monado Cardboard driver, which enables the experience to run on Android phones with Cardboard VR headsets, making the immersive content accessible to a wider audience, including kids and students. This port reinforces our commitment to open standards, allowing for future enhancements and maintaining the project’s long-term sustainability.
+You can find the code for that driver here: https://gitlab.freedesktop.org/bl4ckb0ne/monado/-/tree/cardboard-sdk. It will be upstreamed soon.
```

Comment:
```suggestion
You can find the code for that driver here: [https://gitlab.freedesktop.org/bl4ckb0ne/monado/-/tree/cardboard-sdk](https://gitlab.freedesktop.org/bl4ckb0ne/monado/-/tree/cardboard-sdk). It will be upstreamed soon.
```

Reply:
```suggestion
You can find the code for that driver on the [freedesktop.org GitLab](https://gitlab.freedesktop.org/bl4ckb0ne/monado/-/tree/cardboard-sdk). It will be upstreamed soon.
```